src/CSS/Home.css

```diff
@@ -0,0 +1,37 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.home-wrap {
+  width: 90%;
+  padding: 4rem 2rem;
+  color: #fff;
+  border-radius: 20px;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  background-image: url(../assets/image/crypto.gif);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: initial;
+}
+
+.h2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 1.5rem;
+}
+
```

Comment:
[OPTIONAL]

- [ ] You can work on the design to match what was given in the project requirements.